GNUmakefile

```diff
@@ -50,13 +50,14 @@ gomod: ## Ensure chainlink's go dependencies are installed.
 .PHONY: yarndep
 yarndep: ## Ensure all yarn dependencies are installed
 	yarn install --frozen-lockfile
+	yarn setup
 
 .PHONY: gen-builder-cache
 gen-builder-cache: gomod yarndep # generate a cache for the builder image
 
 .PHONY: install-chainlink
 install-chainlink: chainlink ## Install the chainlink binary.
-	cp $< $(GOBIN)/chainlink
+	cp $< /usr/local/bin/chainlink
```

Comment:
$(GOBIN) surely makes more sense here, will allow this makefile to keep working on windows for instance.

Reply:
Did not think of that, good point.

Reply:
@j16r I also forgot that we've put windows support on hold until we find someone who has more expertise with it. Until then, WSL works.

Reply:
Probably worth mentioning then that `/usr/local/bin` is not even necessarily standard on unix systems.

Reply:
It'd probably be best to have `GOBIN ?= /usr/local/bin` at the beginning of this makefile. So that it can use GOBIN if defined?

Reply:
Ok, I've removed it. We can add this in a separate PR.